staging/src/k8s.io/apimachinery/pkg/runtime/interfaces.go

```diff
@@ -210,6 +210,18 @@ type ObjectCreater interface {
 	New(kind schema.GroupVersionKind) (out Object, err error)
 }
 
+// ResourceMapper provides information about equivalent resources,
```

Comment:
this needs more explanation: resources are GVRs. How can GVRs be equivalent if they are not equal?

Reply:
updated docs